sdk/apis/tenancy/v1alpha1/types_workspace.go

```diff
@@ -195,6 +200,11 @@ type WorkspaceStatus struct {
 	//
 	// +optional
 	Initializers []corev1alpha1.LogicalClusterInitializer `json:"initializers,omitempty"`
+
+	// Mounts is a list of mounts that are mounted into this workspace and their statuses.
+	//
+	// +optional
+	Mounts []MountStatus `json:"mounts,omitempty"`
```

Comment:
this is already the next step, isn't it? If we have mounts here, we don't need the annotation anymore?